edx_sga/static/js/src/edx_sga.js

```diff
@@ -48,13 +48,19 @@ function StaffGradedAssignmentXBlock(runtime, element) {
                                     return;
                                 }
                             }
+                            $(content).find(".upload").attr("role", "progressbar");
+                            $(content).find(".upload").attr("aria-valuemax", "100");
+                            $(content).find(".upload").attr("aria-valuemin", "0");
```

Comment:
A loop would probably be overkill for this, but please store the element before making these calls:
```javascript
var uploadElement =  $(content).find(".upload");
uploadElement.attr(/* params */)
// ...etc...
```